controllers/kyma_controller_helper_test.go

```diff
@@ -23,16 +23,13 @@ import (
 
 func RegisterDefaultLifecycleForKyma(kyma *v1beta1.Kyma) {
 	BeforeAll(func() {
-		Expect(controlPlaneClient.Create(ctx, kyma)).Should(Succeed())
-		DeployModuleTemplates(ctx, controlPlaneClient, kyma)
+		Eventually(controlPlaneClient.Create(ctx, kyma), Timeout, Interval).Should(Succeed())
```

Comment:
Eventually here is given an `error` value produced by invocation of `controlPlaneClient.Create(ctx, kyma)`. It should be given a function that returns an error and internally invokes `controlPlaneClient.Create(ctx, kyma)`.